src/docs/product/releases/index.mdx

```diff
@@ -8,70 +8,88 @@ redirect_from:
 description: "Learn how to provide information to Sentry about your releases to determine regressions, predict the commit that caused the issue, and resolve issues quickly."
 ---
 
-A release is a version of your code that is deployed to an environment. When you give Sentry information about your releases, you unlock a number of new features:
+A _release_ is a version of your code deployed to an environment. When you provide Sentry information about your releases, you can:
 
 - Determine the issues and regressions introduced in a new release
 - Predict which commit caused an issue and who is likely responsible
 - Resolve issues by including the issue number in your commit message
 - Receive email notifications when your code gets deployed
+- Apply [source maps](/platforms/javascript/sourcemaps/) so JavaScript and React Native projects can utilize release artifacts to unminify or symbolicate error stack traces
 
-Additionally, releases are used for applying [source maps](/platforms/javascript/sourcemaps/) to minified JavaScript to view original, untransformed source code.
+After configuring your SDK and notifying Sentry, you can view information about the release, such as new issues and regressions introduced in the release.
 
-## Setting up Releases
+![](releases-overview.png)
 
-Setting up releases fully is a 4-step process:
+## Set Up
 
-1.  [Configure Your SDK](#configure-sdk)
-2.  [Install Repository Integration](#install-repo-integration)
-3.  [Create Release and Associate Commits](#create-release)
-4.  [Tell Sentry When You Deploy a Release](#create-deploy)
+Setting up releases fully requires:
 
-### Configure Your SDK {#configure-sdk}
+1. [Configuring Your SDK](/platform-redirect/?next=/configuration/releases/) to include a release identifier, which is documented for each SDK
```

Comment:
Yes, it's tricky because you don't complete this step here, you should really have done so prior to going into sentry.io to set up the notification process.

Reply:
That's a good point, maybe it's enough to mention it. Even if you'd like to add more content about this, I don't think it has to be new content. We could put just a header "Configuring Your SDK" + the common/set-releases example.